app/src/main/java/com/example/benchproject/app/movie/details/viewModel/MovieDetailsViewModel.kt

```diff
@@ -0,0 +1,36 @@
+package com.example.benchproject.app.movie.details.viewModel
+
+import android.util.Log
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.example.benchproject.app.movie.details.entity.MovieDetails
+import com.example.benchproject.domain.movie.details.entity.mapToUI
+import com.example.benchproject.domain.movie.details.usecase.GetMovieDetailsUseCase
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+@HiltViewModel
+class MovieDetailsViewModel @Inject constructor(
+    private val getMovieDetailsUseCase: GetMovieDetailsUseCase
+) : ViewModel() {
+
+    val responseMovieDetails: LiveData<MovieDetails>
```

Comment:
`response` suffix seems to redundant here

Reply:
Ok, I changed it from `responseMovieDetails` on `movieDetails`